zypp/KeyManager.h

```diff
@@ -0,0 +1,55 @@
+/*---------------------------------------------------------------------\
+|                          ____ _   __ __ ___                          |
+|                         |__  / \ / / . \ . \                         |
+|                           / / \ V /|  _/  _/                         |
+|                          / /__ | | | | | |                           |
+|                         /_____||_| |_| |_|                           |
+|                                                                      |
+\---------------------------------------------------------------------*/
+/** \file	zypp/AutoDispose.h
```

Comment:
done

Reply:
It would be good if you could encode these rules in something consumable by [clang-format](https://clang.llvm.org/docs/ClangFormat.html) , and then have travis run that on every PR/branch.

Reply:
Yes, we already discussed using clang-format as git hook, so commited source code gets auto formatted. However we need to be careful so we do not make backporting too hard with all the whitespace changes. One solution would be to run clang-format on all our branches right away to keep the conflicts to a minimum.